lib/markdown-pdf.js

```diff
@@ -2,7 +2,7 @@ var fs = require("fs");
 var pdf = require('html-pdf');
 var path = require('path');
 var url = require('url');
-var mdpreview = atom.packages.activePackages["markdown-preview"];
+var mdpreview = null // Defer loading untill convert is called
```

Comment:
Deferring the loading of a markdown preview package makes it easier to add support for other preview packages.